src/OrbitGl/CaptureViewElement.cpp

```diff
@@ -248,4 +247,49 @@ void CaptureViewElement::RequestUpdate(RequestUpdateScope scope) {
   }
 }
 
+void CaptureViewElement::PreRender(PrimitiveAssembler& primitive_assembler,
+                                   TextRenderer& text_renderer) {
+  primitive_assembler.PushTranslation(0, 0, DetermineZOffset());
+  text_renderer.PushTranslation(0, 0, DetermineZOffset());
+
+  if (RequestSeparateRenderGroup()) {
+    static const std::string kGroupPrefix = "|rg_cve_";
+    BatchRenderGroupStateManager* manager = primitive_assembler.GetRenderGroupManager();
+
+    previous_batcher_render_group_name_ = primitive_assembler.GetCurrentRenderGroupName();
+    const std::string new_batcher_render_group_name =
+        previous_batcher_render_group_name_ + kGroupPrefix + std::to_string(GetUid());
```

Comment:
What about using `absl::StrFormat("%s%s%u", ...)` in this case? That's more common in our code base, is suggest by the style guide, might be faster, and works with the string_view suggestion from above.

Reply:
I've changed this to `absl::StrCat` now

Reply:
Did you pushed the changes?

Reply:
While we use `StrFormat` quite a lot in the code base, `Substitute` would be probably more appropriate as per [go/totw/18](https://goto.google.com/totw/18). But I guess in the end, all three proposed solutions would be completely fine.

Reply:
Indeed :-) All pushed.